### DISCUSSION TITLE
При создании ресурса добавить опцию сразу создать модель  миграцию

### DISCUSSION THREAD

**Discussion by ArtemYurov** [moonshine-software/moonshine#1258] [Ideas]
При разработке получается так что сразу нужно создавать модель, миграцию и ресурс, было бы удобно делать это одной командой

**Comment by lee-to** [comment]
Мы этого в коробке делать не будем, есть для этого плагины и вот под эту задачу уже есть отличный пакет https://github.com/dev-lnk/moonshine-builder который не только будет генерировать а еще и сразу сделает все на основе бд
